x-pack/plugins/observability_solution/observability/public/pages/alert_details/components/alert_history.tsx

```diff
@@ -162,7 +154,8 @@ export function AlertHistoryChart({ rule, alert }: Props) {
       </EuiFlexGroup>
       <EuiSpacer size="s" />
       <AlertSummaryWidget
-        featureIds={featureIds}
+        ruleTypeIds={OBSERVABILITY_RULE_TYPE_IDS_WITH_STACK}
```

Comment:
Is the idea here that if we are filtering based on the rule, we don't need to limit ruleTypeIds only to the current rule type anymore? Because, previously, we were limiting the featureIds to only the one related to the current alert's rule.